app/src/main/res/xml/content_settings.xml

```diff
@@ -1,120 +1,124 @@
-        app:iconSpaceReserved="false"
         android:defaultValue="false"
         android:key="@string/show_age_restricted_content"
-        android:title="@string/show_age_restricted_content_title"/>
+        android:title="@string/show_age_restricted_content_title"
+        app:iconSpaceReserved="false" />
 
     <SwitchPreferenceCompat
-        app:iconSpaceReserved="false"
         android:defaultValue="false"
         android:key="@string/youtube_restricted_mode_enabled"
-        android:title="@string/youtube_restricted_mode_enabled_title"/>
+        android:title="@string/youtube_restricted_mode_enabled_title"
+        app:iconSpaceReserved="false" />
 
     <SwitchPreferenceCompat
-        app:iconSpaceReserved="false"
         android:defaultValue="true"
         android:key="@string/show_search_suggestions_key"
         android:summary="@string/show_search_suggestions_summary"
-        android:title="@string/show_search_suggestions_title"/>
+        android:title="@string/show_search_suggestions_title"
+        app:iconSpaceReserved="false" />
 
     <SwitchPreferenceCompat
-        app:iconSpaceReserved="false"
         android:defaultValue="true"
         android:key="@string/download_thumbnail_key"
+        android:summary="@string/download_thumbnail_summary"
         android:title="@string/download_thumbnail_title"
-        android:summary="@string/download_thumbnail_summary"/>
+        app:iconSpaceReserved="false" />
 
     <SwitchPreferenceCompat
-        app:iconSpaceReserved="false"
         android:defaultValue="true"
         android:key="@string/show_next_video_key"
-        android:title="@string/show_next_and_similar_title"/>
+        android:title="@string/show_next_and_similar_title"
+        app:iconSpaceReserved="false" />
 
     <SwitchPreferenceCompat
-        app:iconSpaceReserved="false"
         android:defaultValue="true"
         android:key="@string/show_comments_key"
+        android:summary="@string/show_comments_summary"
         android:title="@string/show_comments_title"
-        android:summary="@string/show_comments_summary"/>
+        app:iconSpaceReserved="false" />
 
     <Preference
-        app:iconSpaceReserved="false"
-        android:summary="@string/import_data_summary"
         android:key="@string/import_data"
-        android:title="@string/import_data_title"/>
+        android:summary="@string/import_data_summary"
+        android:title="@string/import_data_title"
+        app:iconSpaceReserved="false" />
 
     <Preference
-        app:iconSpaceReserved="false"
-        android:title="@string/export_data_title"
         android:key="@string/export_data"
-        android:summary="@string/export_data_summary"/>
+        android:summary="@string/export_data_summary"
+        android:title="@string/export_data_title"
+        app:iconSpaceReserved="false" />
+
+    <Preference
+        android:key="@string/clear_cookie_key"
+        android:title="@string/clear_cookie_title"
```

Comment:
Maybe also add a summary, saying something like "if you were asked to solve a reCAPTCHA, permission cookies may have been stored on the device"

Reply:
A summary is to explain what the setting will do. That sentence doesn't explain that.

Reply:
Yeah, but it explains why that setting is there. Otherwise one would see the word "cookie" and go mad at NewPipe thinking it is tracking him, even if he never solved a recaptcha

Reply:
Good point. There should be an explanation.